all-in-one.yaml

```diff
@@ -1,5 +1,97 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
+metadata:
```

Comment:
All ingresses in this file is missing ingress class annotation or the ingress class field. Please add them back.